ci/build_docs.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+# Copyright (c) 2023, NVIDIA CORPORATION.
+
+set -euo pipefail
+
+rapids-logger "Create test conda environment"
+. /opt/conda/etc/profile.d/conda.sh
+
+rapids-dependency-file-generator \
+  --output requirements \
+  --file_key docs \
+  --matrix "cuda=${RAPIDS_CUDA_VERSION%.*};arch=$(arch);py=${RAPIDS_PY_VERSION}" | tee requirements.txt
+
+rapids-mamba-retry create -n test
+conda activate test
+pip install -r requirements.txt
+
+rapids-print-env
+
+rapids-logger "Downloading artifacts from previous jobs"
```

Comment:
```suggestion
```

Doesn't look like any downloads are happening here.

Therefore this line can be removed.